src/main/java/it/grid/storm/Main.java

```diff
@@ -15,6 +17,13 @@ public static void main(String[] args) {
 
     StoRM storm = new StoRM();
 
+    try {
+      storm.init();
+    } catch (BootstrapException e) {
+      log.error(e.getMessage(), e);
+      return;
```

Comment:
Shouldn't it exit with a non-zero exit code here?

Reply:
Fixed with exit(1)